troposphere/static/js/components/projects/detail/resources/tableData/instance/Activity.react.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react/addons';
+import Backbone from 'backbone';
```

Comment:
We're going to be moving away from using `'react/addons'` everywhere because in React >0.14.*, they have split all the _addons_ into their own npm modules. 

This is just a chance for me to mention it to you. You can leave this, I will get it during my mass-changes for Nautical Nighthawk.